bin/update-proxy-versions.sh

```diff
@@ -1,32 +1,32 @@
 #!/usr/bin/env bash
 
 # USAGE:
-# > PROXY_VERSION=1.0.31 ./update-proxy-versions.sh
```

Comment:
Should we rename the script, as well? (Preferably less broken than I did in the JVM SDK.)

Reply:
Yes, lets do that. https://github.com/lightbend/kalix-runtime/pull/2271